requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+python-dateutil
```

Comment:
- python-dateutil is already in setup.py as dependency of czech-holidays
- pytest is needed for testing, so I'd either use tox or directly written `pip install pytest` to `.travis.yml`

I don't want to maintain two lists of dependencies.